packages/crud/src/RecordsList.tsx

```diff
@@ -52,7 +52,10 @@ export class RecordsList extends Component<RecordsListProps> {
   /** this dirty hack needs to avoid passing the old table data after changing the table */
   public getRecordsListData = (recordsListResult: QueryResult<RecordsListData>) => {
     const recordsListData =
-      this.isFetchingNewTable && recordsListResult.loading ? [] : R.path(['data', 'tableContent'], recordsListResult);
+      this.isFetchingNewTable && recordsListResult.loading
```

Comment:
@ej9x please, rewrite this construction with `if-else`. It's hard to read so much logical operators in a row.